Other styles

rondoor124 opened the pull request:
Change styles of `Components` and `Containers` folder files, except for subdirectories.

Demo Link: https://deploy-preview-524--bldrs-share.netlify.app/

`PTAL`

pablo-mayrgundter reviewed (changes requested):
Ok, reabase looks to have worked.  Now for review..

The avatar icon break is back:
<img width="536" alt="image" src="https://user-images.githubusercontent.com/2480879/209450678-0425c0f3-3798-4b54-a903-7debbae89e73.png">


Seeing a new scrollbar in Share dialog:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/2480879/209450660-d6bdebb8-89d0-437a-9ea6-2af911f7b90f.png">

also seeing new warning in the console
<img width="724" alt="image" src="https://user-images.githubusercontent.com/2480879/209450697-c13eabce-2889-4f46-85fd-64fc46315f17.png">

Please fix these up and I'll do a code review.  Thanks!

pablo-mayrgundter commented:
Ok, icons look fixed!  Also not seeing the console error anymore.

New ones:
Scrollbars in mobile:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/2480879/209580779-23840dc2-3a55-44b1-b700-79360a42dff6.png">

lost some spacing in Open dialog:
<img width="246" alt="image" src="https://user-images.githubusercontent.com/2480879/209580833-0f6eee3e-4e13-4154-9633-13352aa7a81a.png">

pablo-mayrgundter commented:
Ok, maybe have a repro for the console warning:

1) Open comments,
2) Open properties,
3) Close properties -> console warning

Complaining about anchorEl

<img width="736" alt="image" src="https://user-images.githubusercontent.com/2480879/209581243-5d56e0f7-678e-4526-88bf-998fe0443b5a.png">

rondoor124 commented:
Hi @pablo-mayrgundter.
I addressed all issues you mentioned, please review.
the e2e test works well locally, but get error occasionally on live. seems due to cypress disadvantage.
please let me know the next steps. thank you.

pablo-mayrgundter commented on src/utils/itemProperties.jsx:
```diff
@@ -1,6 +1,6 @@
 import React, {useState} from 'react'
 import debug from './debug'
-import Typography from '@mui/material/Typography'
+import {Typography} from '@mui/material'
```
Revert

rondoor124 replied:
I have done.

pablo-mayrgundter commented on src/Components/Toggle.jsx:
```diff
@@ -1,5 +1,5 @@
 import React, {useContext} from 'react'
-import Switch from '@mui/material/Switch'
+import {Switch} from '@mui/material'
```
Revert

rondoor124 replied:
I have done.

pablo-mayrgundter commented on src/Components/SnackbarMessage.jsx:
```diff
@@ -1,6 +1,6 @@
 import React, {useContext} from 'react'
+import {Snackbar} from '@mui/material'
 import MuiAlert from '@mui/material/Alert'
-import Snackbar from '@mui/material/Snackbar'
```
Revert

rondoor124 replied:
I have done.

pablo-mayrgundter commented on src/Components/SideDrawer/SideDrawerPanels.jsx:
```diff
@@ -1,7 +1,5 @@
-import Box from '@mui/material/Box'
-import Typography from '@mui/material/Typography'
-import {makeStyles, useTheme} from '@mui/styles'
+import {Box, Typography} from '@mui/material'
```
Revert

rondoor124 replied:
I have done.

pablo-mayrgundter reviewed (approved):
Sorry, just noticed you've moved from default to named imports for all the mui components.  Please move back to defaults as per this note:

Ihttps://mui.com/material-ui/guides/minimizing-bundle-size/

We've tested this out and it does save a few hundred KB on the bundle.

I test locally for any named imports with:

```
> grep -r \'@mui/material\' src
src/Components/Dialog.jsx:import {Typography} from '@mui/material'
src/Components/AboutControl.jsx:import {Box} from '@mui/material'
```

And then edit them back to default.

rondoor124 commented:
@pablo-mayrgundter, I reverted all import method to the original status.
Please check the update, and give me feedback.
Thank you. :)
